StudentsMVC/WebContent/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
Please check if this manifest file is really needed in the commit or not (i.e. if it is auto-generated by Spring or not). If it is auto-generated, delete it.

Reply:
It was autogenerated, I've removed it.